### DISCUSSION TITLE
Issue : Uncaught ReferenceError: this hasn't been initialised - super() hasn't been called

### DISCUSSION THREAD

**Discussion by Anushan20** [dolanmiu/docx#1896] [Q&A] [answered]
Hello, 

I am experiencing an issue when adding text to TextRun.

I have the following problem : Uncaught ReferenceError: this hasn't been initialised - super() hasn't been called
![image](https://user-images.githubusercontent.com/55843818/213916251-84170650-a95e-4782-8699-3d6770bee8f9.png)

The code is the basic example :
![image](https://user-images.githubusercontent.com/55843818/213916640-6a32db47-38ad-4a09-9b7a-c3c3627b1c29.png)

When i put empty text i don't have this problem, like :  new TextRun("")

The issue is present on my server, but it works fine on my localhost and I am using react.

This is my first time working with this library, so I might be missing something obvious. Any help would be greatly appreciated!

**Comment by dolanmiu** [comment, marked as answer]
What version of `docx` are you using?

**Reply by Anushan20** [reply]
It's 7.8.0.

**Reply by dolanmiu** [reply]
Try the latest version, I have fixed this in `7.8.2`

**Reply by Anushan20** [reply]
Thank you so much it worked !!!
